src/includes/traits/Plugin/OptionUtils.php

```diff
@@ -62,6 +62,8 @@ public function updateOptions(array $options)
         $this->options = array_intersect_key($this->options, $this->default_options);
         update_site_option(GLOBAL_NS.'_options', $this->options);
 
+        file_put_contents(WP_CONTENT_DIR.'/debug.log', print_r(get_defined_vars(), true)."\n\n", FILE_APPEND);
```

Comment:
@kristineds Please remove this line before submitting a PR. This is for _local_ debugging only and should never be included in any merge into the dev branch where it could end up on a live site and create a huge log file over time.